ebpf/cmd/playground/main.go

```diff
@@ -3,57 +3,153 @@
 package main
 
 import (
+	"bytes"
+	"context"
 	"fmt"
 	"os"
-	"strings"
 	"time"
 
+	"github.com/bufbuild/connect-go"
 	"github.com/go-kit/log"
+	"github.com/grafana/pyroscope/ebpf/metrics"
+
+	"github.com/go-kit/log/level"
+	pushv1 "github.com/grafana/pyroscope/api/gen/proto/go/push/v1"
+	"github.com/grafana/pyroscope/api/gen/proto/go/push/v1/pushv1connect"
+	typesv1 "github.com/grafana/pyroscope/api/gen/proto/go/types/v1"
 	ebpfspy "github.com/grafana/pyroscope/ebpf"
+	"github.com/grafana/pyroscope/ebpf/pprof"
 	"github.com/grafana/pyroscope/ebpf/sd"
 	"github.com/grafana/pyroscope/ebpf/symtab"
 	"github.com/grafana/pyroscope/ebpf/symtab/elf"
 	"github.com/prometheus/client_golang/prometheus"
+	commonconfig "github.com/prometheus/common/config"
 )
 
+const sampleRate = 99 // times per second
+
+// todo remove this cmd or move to a separate go module or out of repository, because it adds a bunch of dependencies
```

Comment:
I'd love a CLI to activate the profiler locally.

Reply:
this main file is exactly that.
The comment is about that this cli is using more dependencies that is required (for example grpc)
Im going to just remove this comment until we have someone complain